README.md

```diff
@@ -1,16 +1,20 @@
-# 💊 ReVanced Manager
+<p align="center">
+  <img width="800" src="Logo (with text).png">
+</p>
```

Comment:
The logo can be improved by changing the font to [Manrope](https://github.com/sharanda/manrope "Manrope font on GitHub"), Open Sans, or similar. See my example design for reference. *you can use this design, there's nothing stopping you :) but [see this.](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to)*

![ReVanced Manager Valentine - Light mode](https://user-images.githubusercontent.com/93124920/218297719-bceb269d-8abc-4007-be5e-59c756513b48.png "ReVanced Manager banner - Valentine themed - Light mode")


![ReVanced Manager Valentine - Dark mode](https://user-images.githubusercontent.com/93124920/218297710-30f88511-fa37-4c93-b37b-42438ef58829.png "ReVanced Manager banner - Valentine themed - Dark mode")

Reply:
Why font needs to change? 
Why not follow the brand guide?

Reply:
![revanced-manager-wordmark](https://user-images.githubusercontent.com/82371061/218303155-28b9df3e-9c74-4f34-b70c-247af9586695.svg)

Reply:
We currently lack an extensive kit of our current language design. A repository should be created to discuss this matter openly.

Reply:
> Why font needs to change? Why not follow the brand guide?

I just realize Manrope is the font used on ReVanced wordmark. My bad